### PR TITLE
Support multiple date formats for Serializer

### PR DESCRIPTION
**Pull request by armando-rodriguez-cko**
## Support multiple date formats for Serializer
- Adds strings format for the correct deserialisation from json responses
- Adds unit tests

### CODE REVIEW

File: test/CheckoutSdkTest/JsonSerializerTest.cs [hunk 1]
```diff
@@ -112,5 +113,47 @@ public void ShouldDeserializeDefaultCardCTypeResponse()
...
+                "{\"processed_on\":\"2021-06-08T12:25:01.0000000\"}",
+            };
+
+            foreach (var date in dates)
+            {
+                PaymentResponse paymentResponse =
+                    (PaymentResponse)new JsonSerializer().Deserialize(date, typeof(PaymentResponse));
+
+                paymentResponse.ShouldNotBeNull();
+                paymentResponse.ProcessedOn.ShouldNotBeNull();
```
Comment (armando-ibarra-cko) [inline test/CheckoutSdkTest/JsonSerializerTest.cs:155]: We need at least assert that most of the dates have `2021-06-08T12:25:01` to make sure it does not change anything

Comment (armando-rodriguez-cko) [reply test/CheckoutSdkTest/JsonSerializerTest.cs:155]: 👍🏼

File: test/CheckoutSdkTest/Disputes/Previous/DisputesIntegrationTest.cs [hunk 2]
```diff
@@ -11,9 +11,10 @@ public class DisputesIntegrationTest : AbstractPaymentsIntegrationTest
         [Fact]
         private async Task ShouldQueryDisputes()
         {
+            //TODO format
```
Comment (martinseco) [inline test/CheckoutSdkTest/Disputes/Previous/DisputesIntegrationTest.cs:14]: What about this TODO? can we solve it now?

Comment (armando-rodriguez-cko) [reply test/CheckoutSdkTest/Disputes/Previous/DisputesIntegrationTest.cs:14]: Sorry, mistake, I have deleted it

Comment (armando-rodriguez-cko) [reply test/CheckoutSdkTest/Disputes/Previous/DisputesIntegrationTest.cs]: Thank you!

**Review by martinseco — APPROVED** [review]
LGTM

**Pull request merged**